api/services.py

```diff
@@ -35,3 +35,45 @@ def decorator(*args, **kwargs):
 
 def get_search_value(request):
     return request.args.get('search') if request.args.get('search') else ''
+
+
+def get_sort_parametr(request):
```

Comment:
```suggestion
def get_sort_parameter(request):
```
the same for the usages of the word below